hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieBackedMetadata.java

```diff
@@ -2482,6 +2483,14 @@ public void testMetadataMetrics() throws Exception {
     }
   }
 
+  @Test
+  public void testGetFileGroupIndexFromFileId() {
```

Comment:
I meant for all similar static methods
getFileIDForFileGroup
getFileGroupIndexFromFileId
getFileGroupPrefix
getFileIdLengthWithoutFileIndex

Reply:
Added test for first 3. 

getFileIdLengthWithoutFileIndex is private and is called from other functions so is indirectly tested.